src/contact.js

```diff
@@ -0,0 +1,99 @@
+const ham = document.querySelector('.hamburger');
+const menu = document.querySelector('.menu');
+const logo = document.querySelector('.logo');
+const lists = document.querySelectorAll('.list');
+
+ham.addEventListener('click', () => {
+  ham.classList.toggle('active');
+  menu.classList.toggle('in-active');
+  logo.classList.toggle('in-active');
+});
+
+logo.addEventListener('click', () => {
+  ham.classList.toggle('active');
+  menu.classList.toggle('in-active');
+  logo.classList.toggle('in-active');
+});
+
+const spreadList = [...lists];
+spreadList.forEach((list) => {
+  list.addEventListener('click', () => {
+    ham.classList.toggle('active');
+    menu.classList.toggle('in-active');
+    logo.classList.toggle('in-active');
+  });
+});
+
```

Comment:
- [OPTIONAL] Those lines of code is duplicated (you have exactly the same code in main.js file), So I suggest deleting this part of code from here to make your code clean and DRY👌
And in your `contact.html` file, You can use more than on script tag
```
<script src="./src/main.js"></script>
<script src="./src/contact.js"></script>
```
> To follow [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) and to Keep your [code clean](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices) 💯